pkg/workflows/wasm/pb/wasm.proto

```diff
@@ -74,7 +70,13 @@ message FetchRequest {
 message FetchResponse {
   bool success = 1;
   string errorMessage = 2;
-  uint32 statusCode = 3; // NOTE: this is actually a uint8, but proto doesn't support this.
+  uint32 statusCode =
+      3; // NOTE: this is actually a uint8, but proto doesn't support this.
   values.Map headers = 4;
   bytes body = 5;
 }
+
+message CustomEmitMessage {
```

Comment:
Consider just calling this `EmitMessage` -- CustomEmitMessage doesn't say much more than that.